src/packages.builds

```diff
@@ -31,7 +31,7 @@
           BeforeTargets="BuildAllProjects"
           Condition="'$(IncludePreReleaseLabelInPackageVersion)' != 'true'">
     <ItemGroup>
-      <PkgProjects Include="$(MSBuildThisFileDirectory)..\pkg\*\*.pkgproj" />
+      <PkgProjects Include="$(MSBuildThisFileDirectory)..\pkg\*\*.pkgproj" Exclude="$(MSBuildThisFileDirectory)..\pkg\*Private*\*.pkgproj" />
```

Comment:
This will exclude more than just the two erroring pkgproj's (there were one or two that didn't fail).  Is this just a workaround for now?